backend/BackendMIIT/src/main/java/com/example/BackendMIIT/service/ProfileService.java

```diff
@@ -10,4 +9,12 @@ public interface ProfileService {
     void parseProfile(String url);
 
     void saveProfile(List<String> properties);
+
+    List<ProfileDto> getProfilesByInstitute(String institute);
+
+    List<ProfileDto> getAllProfiles();
+
+    ProfileDto getProfileByName(String name);
+
+    List<ProfileDto> getProfilesByDirection(String code);
```

Comment:
Recommend to rename `getProfilesByDirection` to `getProfilesByDirectionCode`

Reply:
Renamed